### DISCUSSION TITLE
Customizing Pluto launch

### DISCUSSION THREAD

**Discussion by Vjust** [fonsp/Pluto.jl#764] [Questions about using Pluto]
Is there a config file to customize Pluto?. I am looking to specify the "launch_browser" parameter, and persist the same in a config file.

**Comment by fonsp** [comment]
No, but you can write a custom launch function/script

**Reply by fonsp** [reply]
For example, you could add this to your REPL init script (I forgot what it's called, sorry)
```julia
function run_my_pluto()
    Core.eval(Main, quote
        import Pkg
        Pkg.update("Pluto")
        import Pluto
        Pluto.run(launch_browser=false)
    end)
end
```

**Comment by fonsp** [comment]
FYI, you can read the list of (advanced) configuration options by giving false arguments: `Pluto.run(asdfasdf=123)`

**Comment by Vjust** [comment]
I misunderstood the "launch_browser" purpose.. On my system it launches Chrome (default), but I wanted it to launch Firefox.. I thought this configuration parameter "launch_browser" would help me do that. Thanks for clarifying.
